build/Publish.targets

```diff
@@ -73,6 +74,10 @@
         <ManifestArtifactData>ShipInstaller=dotnetcli</ManifestArtifactData>
       </FilesToPublish>
 
+      <SymbolPackagesToPublish Include="$(DependencySymbolsDir)\**\*.symbols.nupkg" />
```

Comment:
I think at this point, the item groups can be reduced to one since we don't need to transform metadata anymore.